<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Babel - Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Assets/images/favicon.png</normaloff>../../../Assets/images/favicon.png</iconset>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftBar">
      <item>
       <layout class="QVBoxLayout" name="userInfosLayout">
        <item>
         <widget class="QLabel" name="usernameLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Hello &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;John Doe,&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ipAddressLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Your ip address is &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;10.0.0.42.1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="contactOnlineLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;There are &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;0&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; contacts online.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="dividerLayout">
        <item>
         <widget class="Line" name="dividerLIne">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="contactsListLayout">
        <item>
         <widget class="QLabel" name="contactsLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Contacts &amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="contactList">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignJustify|AlignVCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IP Address</string>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignJustify|AlignVCenter</set>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightBar">
      <item>
       <layout class="QVBoxLayout" name="callLayout">
        <item>
         <widget class="QLabel" name="callInfosLabel">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#484848;&quot;&gt;No phone call infos to display...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="callActionsLayout">
          <item>
           <widget class="QPushButton" name="callButton">
            <property name="text">
             <string>Call</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Assets/images/call.png</normaloff>../../../Assets/images/call.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hangOutButton">
            <property name="text">
             <string>Hang out</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Assets/images/hangout.png</normaloff>../../../Assets/images/hangout.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>callButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>call()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hangOutButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>hangOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>706</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>call()</slot>
  <slot>hangOut()</slot>
  <slot>addContact()</slot>
  <slot>editContact()</slot>
  <slot>deleteContact()</slot>
 </slots>
</ui>
